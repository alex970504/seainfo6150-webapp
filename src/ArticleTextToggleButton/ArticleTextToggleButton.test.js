import React from "react";
import ArticleTextToggleButton from "./ArticleTextToggleButton";
import { fireEvent } from "@testing-library/react";
import ArticleImage from "../ArticleImage/ArticleImage";

describe("ArticleListItem tests", () => {
  test("Click", () => {
    const onclick = () => {console.log("clicked")};
    const {container} = render(<ArticleTextToggleButton onclick={onclick}/>);
    expect(container).toMatchSnapshot();
  });
});