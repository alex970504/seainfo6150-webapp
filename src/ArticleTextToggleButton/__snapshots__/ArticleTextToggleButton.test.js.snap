// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleListItem tests Click 1`] = `
<div>
  <button
    class="button"
  />
</div>
`;
